projects/aws-observability/aws-otel-collector/Makefile

```diff
@@ -0,0 +1,41 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+HELM_GIT_TAG:=$(shell cat HELM_GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REGISTRY=public.ecr.aws/aws-observability
+COLLECTOR_IMAGE=aws-otel-collector
+
+REPO_OWNER=aws-observability
+REPO=aws-otel-collector
+
+SIMPLE_CREATE_BINARIES=false
+SIMPLE_CREATE_TARBALLS=false
+HAS_LICENSES=false
+IMAGE_NAMES=
+
+BUILD_TARGETS=helm/build
```

Comment:
Can we avoid these two overrides? Without this what targets are run?

Reply:
I believe they are necessary. See screenshot below without the override.

<img width="797" alt="Screen Shot 2022-08-26 at 9 31 15 AM" src="https://user-images.githubusercontent.com/19214133/186928807-9b8bb143-2519-4cd2-919a-dcb3fba6c2a6.png">

Reply:
Ah yeah i see its because the helm targets are added to the IMAGE_TARGETS, which I want to change. this is fine for now.